app/views/collection_associations/new.html.erb

```diff
@@ -3,8 +3,14 @@
     <h2><%= current_user.provider_id %> <%= resource_name %> Collection Association Search</h2>
     <%= render partial: 'cmr_search/shared/search_form', locals: { submit_path: send("new_#{lower_resource_name}_collection_association_path", @concept_id) } %>
 
-    <p><span class="fa fa-info-circle"></span> Disabled rows in the results below represent collections that are already associated with this <%= lower_resource_name %>.</p>
+    <% if not_variable? %>
+      <p><span class="fa fa-info-circle"></span> Disabled rows in the results below represent collections that are already associated with this <%= lower_resource_name %>.</p>
 
-    <%= render partial: 'cmr_search/shared/search_results', locals: { collections: @collections, ignored_concept_ids: @previously_associated_collections, submit_path: send("#{lower_resource_name}_collection_associations_path", @concept_id), submit_text: 'Submit' } %>
+      <%= render partial: 'cmr_search/shared/search_results', locals: { collections: @collections, ignored_concept_ids: @previously_associated_collections, submit_path: send("#{lower_resource_name}_collection_associations_path", @concept_id), submit_text: 'Submit' } %>
+    <% else %> 
+      <p><span class="fa fa-info-circle"></span> Variables can only be associated with a single collection.</p>
+
+      <%= render partial: 'cmr_search/shared/search_results', locals: { collections: @collections, ignored_concept_ids: @previously_associated_collections, submit_path: send("#{lower_resource_name}_collection_associations_path", @concept_id), submit_text: 'Submit', use_radio_buttons: true } %>
```

Comment:
I think you can just call the render partial once, and use `variable?` or `not_variable?` for the value for `use_radio_buttons`